### DISCUSSION TITLE
Viewing RTSP Streams from IP Cameras

### DISCUSSION THREAD

**Discussion by kbr-asun** [ant-media/Ant-Media-Server#6279] [Q&A] [answered]
Hi I am currently on the community version of ant media server and got a test input from OBS working. I have been trying to get a stream source from https://rtsp.stream/admin/dashboard working as a test for our ip cameras and dev purposes. This stream seems to work in VLC but ant media seems to be stuck in a perpetual state of preparing? I was hoping I would be able to use the embedded player to view this stream source or is that not possible in the community version.

Ive read the docs mentioned here https://github.com/orgs/ant-media/discussions/4986

**Comment by yashtandon113** [comment, marked as answer]
HI @kbr-asun 
Thanks for sharing the link to get the testing RTSP feed. 

I added the RTSP URL as a stream source and it seems to be working fine. Please add the stream with stream source.

Check out [this document](https://antmedia.io/docs/guides/publish-live-stream/ip-cameras-and-external-resources/#re-streaming-external-sources) to add the RTSP link as an external stream source.

**Reply by kbr-asun** [reply]
Thanks for checking it up for me, I saw that link earlier and now believe it may be an issue with our corporate firewalls as i managed to get our cameras connected with stream sources as well as other RTSP test streams. Not sure why these particular ones weren't working for us. Thanks again for the help.

**Reply by kbr-asun** [reply]
The documentation mentioned taking in a variety of inputs does that hold true for the community version as well or just the enterprise?

**Reply by yashtandon113** [reply]
@kbr-asun 
Yes, you can use RTSP, RMTP, HLS, etc. as stream sources in both the community and enterprise editions as well.
